viewer/core/src/public/migration/types.ts

```diff
@@ -16,6 +16,23 @@ export type Color = {
   g: number;
   b: number;
 };
+
+/**
+ * Units supported by {@link Cognite3DModel}.
+ */
+export type WellKnownUnit =
```

Comment:
Rename to something like MeasurementUnit or something similar? WellKnownUnit seems very subjective, well known to who?

Reply:
I think that `WellKnown` is a quite well established naming scheme to mark that it's stuff that is well known to a certain API. Some examples:
https://developers.google.com/protocol-buffers/docs/reference/csharp/class/google/protobuf/well-known-types/enum
https://docs.microsoft.com/en-us/dotnet/api/system.security.principal.wellknownsidtype?view=net-5.0
https://docs.microsoft.com/en-us/dotnet/api/?term=WellKnown